ipywidgets/widgets/widget.py

```diff
@@ -380,12 +380,14 @@ def hold_sync(self):
         """Hold syncing any state until the context manager is released"""
         # We increment a value so that this can be nested.  Syncing will happen when
```

Comment:
This looks like a smart way to do this.  This comment should be updated, though.